acknowledge_license_cert.html

```diff
@@ -267,10 +267,10 @@
 <tr>
 <td>
 Open Med Practice</td>
-<td><a href="http://ww2.openmedpractice.com/">ww2.openmedpractice.com</a><br>
-USA<br>
+<td><a href="http://www.openmedpractice.com/">www.openmedpractice.com</a><br>
+505 Independance Parkway, Chesapeake, VA 23320<br>
 Phone: +1 757 328 2736<br> 
-Sherwin Gaddis: sgaddis@jse.net</td>
+Sherwin Gaddis: sherwin@openmedpractice.com</td>
 </tr>
```

Comment:
Note I bring in any changes or new entries on the above wiki page into the acknowledge_license_cert.html script (this ensure they are the same).